### ISSUE TITLE
[FAST-49] Title of the page doesn't have 'topcoder' in it. It should be 'Community - Accessibility Testing | Topcoder'.

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3105]
**Steps to Reproduce**
1. Launch the url https://www.topcoder.com/community/accessibility-testing.
 2. Ensure the screen reader is on. 
3. Press the combination 'insert+t' to read the title of the page. 
4. Note that the name of the company is not a part of the title of the page.

**Expected Result**
The name of the company 'topcoder' should be part of the page title.

**Actual Result**
The name of the company 'topcoder' is not a part of the page title.

**Device**: Desktop/Labtop
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.2 Page Titled
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=d201e9b0-60ca-4f87-97bc-5041dbbfc8f1&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft-copilots_topcodermsft_onmicrosoft_com%2FDocuments%2F[FAST] Test for Accessibility Series%2Fkameshwarid%2Fbug8

**Comment by hokienick** [comment]
This has been updated in Contentful
